python/cuda_cccl/cuda/cccl/parallel/experimental/_cccl_interop.py

```diff
@@ -299,9 +300,15 @@ def set_cccl_iterator_state(cccl_it: Iterator, input_it):
 
 
 @functools.lru_cache()
-def get_paths() -> List[str]:
-    paths = [f"-I{path}" for path in get_include_paths().as_tuple() if path is not None]
-    return paths
+def get_includes() -> List[str]:
```

Comment:
Probably not worth running the CI again for this, but `get_include_options` would be nice.